aten/src/ATen/native/quantized/cpu/onednn_utils.h

```diff
@@ -0,0 +1,154 @@
+#pragma once
+
+#include <ATen/Config.h>
+#if AT_MKLDNN_ENABLED()
+#include <ATen/Tensor.h>
+#include <ATen/native/quantized/cpu/conv_packed_params.h>
```

Comment:
Hi @Xia-Weiwen, I landed the PR but looks like this line is not up to date. we should remove this line. I'm reverting the change right now, can you help recreate the PR after this is reverted?

Reply:
Hi @jerryzh168, I created a new PR https://github.com/pytorch/pytorch/pull/74137. Please take a look. Thanks.